woocommerce/wceu23-shipping-workshop-final
# Fix code not running on non-block requests

## What

Fixes the example code to work with latest store API changes, specifically:

Move WC core hooks (e.g. order confirmation/order admin/email hooks) into the `Shipping_Workshop_Extend_Woo_Core` class.

Fixes some lint issues.

Fixes #3

## Why

To ensure the example is working well for third party developers.



## Testing Instructions



1. Download the repo as a zip file on this branch or check out this branch.
2. Install on to your site and activate it.
3. Go to the Checkout block. Ensure you see the `IF I AM NOT AT HOME PLEASE…` section.
4. Change the options ensure it works.
5. Check out.
6. Ensure the option you chose is shown on the order confirmation page.
7. Check the order email, ensure the option is in there for both admin and shoppers.
8. Check the order in the WC Back end. Ensure the shipping instruction is there too.
9. Ensure no warnings or errors appear during testing.
